specifications/logging/default.yaml

```diff
@@ -0,0 +1,45 @@
+# this is a default logging configuration of cli
+
+# global filter; messages must have at least this severity to pass:
+minSeverity: Debug
+
+# here we set up outputs of logging in 'katip':
+setupScribes:
+  - scKind: StdoutSK
+    scFormat: ScText
+    scName: text
+  - scKind: StdoutSK
+    scFormat: ScJson
+    scName: json
+
+# if not indicated otherwise, then log output is directed to this:
+defaultScribes:
+  - - StdoutSK
+    - text
+
+# this is probably a bug of parsing, cannot enforce cgBindAddrPrometheus = Nothing
```

Comment:
this is something I cannot go through : 
so when I put `hasPrometheus:` or when I omit to specify this option I end up with: 
```
cgBindAddrPrometheus = Just ("127.0.0.1",12799)
```
When I `hasPrometheus: ["",0]` I end up with
```
cgBindAddrPrometheus = Just ("",0)
```
which is wrong (it should be validated). And it is expected to have `cgBindAddrPrometheus = Nothing`

Reply:
I'd suggest to open an issue on the iohk-monitoring-framework. Fixing this is not part of our scope, but reporting it is :)